apps/fm/js/fm.js

```diff
@@ -918,11 +918,18 @@ function init() {
   );
 }
 
-window.addEventListener('load', function(e) {
-  AirplaneModeHelper.ready(function() {
-    airplaneModeEnabled = AirplaneModeHelper.getStatus() == 'enabled';
-    init();
-
+navigator.mozL10n.once(function(e) {
```

Comment:
No, I'm saying two things:

 - here, I think you need the `localized` event instead of `mozL10n.once`.  While the latter only means that the `ctx` is ready to be used, the former also notifies us that the DOM has been translated and is ready to be displayed.

 - we don't have a `once()`-like method for the `localized` event.  You expressed concern that any `addEventListener` could run after the event fired.  If I understans our code right, as long as you call it in the same event loop iteration that l10n.js has been executed in  you're safe.  `localized` is fired in one of the following event loop iterations because the launch logic of l10n.js is async: `getAdditionalLanguages` and IO make it so.